CHANGELOG.md

```diff
@@ -3,6 +3,71 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.6.0
+
+_Jun 1, 2023_
+
+We'd like to offer a big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- Add `DigitalClock` to [`DesktopDateTimePicker`](https://mui.com/x/react-date-pickers/date-time-picker/)
```

Comment:
Maybe we could add a video or a screenshot? cc @LukasTy

Reply:
```suggestion
- 🚀 New date time picking UI on [`DesktopDateTimePicker`](https://mui.com/x/react-date-pickers/date-time-picker/)

  <img src="https://user-images.githubusercontent.com/4941090/242533399-2b106390-8158-4aba-9ca4-b621c6310977.gif" width="628" />
```

Reply:
Maybe use video for better quality?

Reply:
Alright, no problem